cloud/ec2.go

```diff
@@ -250,9 +253,33 @@ func (m *ec2Manager) Configure(ctx context.Context, settings *evergreen.Settings
 		m.region = evergreen.DefaultEC2Region
 	}
 
+	role, err := getRoleForAccount(settings, m.account)
+	if err != nil {
+		return errors.Wrap(err, "getting role for account")
+	}
+	m.role = role
+
 	return nil
 }
 
+func getRoleForAccount(settings *evergreen.Settings, account string) (string, error) {
+	if account == "" {
```

Comment:
What's the purpose of this check? If there's no account associated there should be no role associated?